app/sdil/models/variations/VariationData.scala

```diff
@@ -32,10 +34,25 @@ case class VariationData(original: RetrievedSubscription,
                          importsVol: Option[Litreage],
                          updatedProductionSites: Seq[Address],
                          updatedWarehouseSites: Seq[Address], // TODO create variation Site model with trading name
-                         updatedContactDetails: ContactDetails
+                         updatedContactDetails: ContactDetails,
+                         previousPages: Seq[Call]
                         )
 
 object VariationData {
+  implicit val callWrites: Writes[Call] = new Writes[Call] {
```

Comment:
you can use the macro to create a `Format[Call]` in one line - see the `VariationData` format